### ISSUE TITLE
A11y: Screenreader announces incorrrect information for the div embedded inside the webview

### ISSUE THREAD

**Issue by vidyasiva** [electron/electron#32146]
Operating system : Mac

Case: div is embedded inside the webview and with role=gridcell. Div contains no data(empty)

Expected Behaviour:
voiceover should announce 'you are currently in the group, to exit this group press Control+Option+Shift+Up arrow '

Actual Behaviour:
Voiceover announces incorrect information as 'you are currently in the group, press Control+Option+Shift+Down arrow to interact with items'

If not inside electron and if it is directly invoked from the browser voiceover announces as expected
